client/src/components/categories.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react';
+
+
+class Categories extends Component {
```

Comment:
Can you change the components to be a Functional components (just a function that returns the jsx) .. there is no need to use `class` if you're not updating states. Also by convention, the file names of the components should capitalized (so *Categories.js* not *categories.js*)